File: Primo.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Primo {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        List<Integer> primo = new ArrayList<>();

        System.out.print("Digite um numero: ");
        int num1 = sc.nextInt();
        sc.close();

        for (int i = num1; i > 0; i--) {
            if (num1 % i == 0) {
                primo.add(i);
            }
        }

        if (primo.size() == 2) {
            System.out.println("O numero " + num1 + " eh primo, pois eh dividido apenas por " + primo + ".");
        } else {
            System.out.println("O numero " + num1 + " nao eh primo, pois eh dividido por " + primo + ".");
        }
    }
}

File: primo.js
//identifique se o número é primo ou não

function verificarPrimo() {
    const num1 = parseInt(document.getElementById("numero").value);
    const primo = [];

    for (let i = num1; i > 0; i--) {
        if (num1 % i == 0) {
            primo.push(i);
        }
    }

    if (primo.length == 2) {
        document.getElementById("resultado").textContent = `O número ${num1} é primo, pois é dividido apenas por ${primo}.`;
    }
    else {
        document.getElementById("resultado").textContent = `O número ${num1} não é primo, pois é dividido por ${primo}.`;
    }
}